lib/time/time.go

```diff
@@ -10,16 +15,54 @@ import (
 	"go.starlark.net/syntax"
 )
 
-// Module time is a Starlark module of time-related functions.
+// Module time is a Starlark module of time-related functions and constants.
+// The module defines the following functions:
+//
+//     from_timestamp(sec, nsec) - Converts the given Unix time corresponding to the total amount of seconds
+//                                 and nonoseconds since January 1, 1970 UTC into an object of type Time.
+//                                 The total amount of nonoseconds is optional.
+//                                 It is valid to pass nonoseconds outside the range [0, 999999999].
+//                                 Not all sec values have a corresponding time value. One such
+//                                 value is 1<<63-1 (the largest int64 value).
+//
+//     is_valid_timezone(loc) - Indicates whether the given name of location is valid or not.
```

Comment:
"is_valid_timezone(loc) - Reports whether loc is a valid time zone name."

(And delete the second sentence.)